portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,90 @@
+        <p>
+          Hey guys, my name is Utibeno Ekpoudom, but Uty is fine as well. I am
+          currently an undergrad studying Computer Science at Georgia Tech.
+          While my concentrations are in intelligence and Infonetworks, I do
+          appreciate amazing design, especially when it comes to web
+          development. Therefore, In addition to my core concentrations I do
+          enjoy designing visually stunning website UIs. Apart from that, I do
+          enjoy watching TV Show, Altered Carbon, GoT, The Expanse etc. As you
+          can tell by my examples I love SciFi, be it shows or movies.
+        </p>
+      </div>
+      <div class="gallery">
+        <img src="images/gatech.jpg" alt="gatech" />
+        <img src="images/nigerianFlag.jpg" alt="" />
+        <img src="images/bugattiDivo.jpg" alt="" />
+        <img src="images/gameOfThrones.jpg" alt="" />
+        <img src="images/tonyStark.jpg" alt="" />
+        <img src="images/alteredCarbon.jpg" alt="" />
+      </div>
+    </section>
+    <section class="project">
+      <div class="paragraph">
+        <p>
+          As for projects, I have worked on some websites inorder to improve my
+          web dev skills, be it in traditional HTML/CSS or through a framework
+          like REACT. I've also dabbled with some backend as well with Node.Js,
+          MongoDB and API creation. To synchronize all these mini projects, I am
+          working on creating a website that provides information on cars mostly
+          supercars, because I do love my supercars (Nah, I don't have one yet
+          but soon😊).
+        </p>
+      </div>
+      <div class="gallery">
+        <img src="images/project1.png" alt="" />
+        <img src="images/react.png" alt="" />
+        <img src="images/nlp.png" alt="" />
+      </div>
+    </section>
+    <section class="places">
+      <div class="paragraph">
+        <p>
+          I do love travelling and trying new things with friends and strangers
+          (potential friends)!
+        </p>
+      </div>
+      <div class="gallery">
+        <img src="images/image1.jpg" alt="great-wall with uty" />
+        <img src="images/image7.jpg" alt="hong-kong" />
+        <img src="images/image3.jpg" alt="thai- temple" />
+        <img src="images/image4.jpg" alt="reclining buddha" />
+        <img src="images/image5.jpg" alt="thailand-temple" />
+        <img src="images/image6.jpg" alt="great-wall with friends" />
+      </div>
+    </section>
+    <div class="talent">
+      <h2>Hidden Talent</h2>
+      <img src="images/arsenal.jpg" alt="hidden talent: soccer" />
     </div>
+    <footer>
+      <a href="https://github.com/utibeno"
+        ><img src="https://img.icons8.com/bubbles/50/000000/github.png"
+      /></a>
+      <a href="https://www.linkedin.com/in/utibenoekpoudom/"
+        ><img src="https://img.icons8.com/bubbles/50/000000/linkedin.png"
+      /></a>
+    </footer>
   </body>
-</html>
+</html>
```

Comment:
nit: this red "circle slash + carriage return" means that the line doesn't end in a trailing newline. It's good practice for all text files to end with a trailing newline, it tends to work better with lots of tools.

Reply:
Gotcha, so I should just add an extra line to make it 91?